hazelcast/src/main/resources/hazelcast-config-3.8.xsd

```diff
@@ -1172,6 +1173,23 @@
             </xs:simpleType>
         </xs:attribute>
     </xs:complexType>
+
+    <xs:complexType name="lock">
```

Comment:
Consider adding support for serializing the new elements in `ConfigXmlGenerator` (but feel free to add it later with a separate PR, it's not crucial).